Components/list/list.css

```diff
@@ -0,0 +1,49 @@
+.list-container{
+    margin: 2rem 2rem 2rem 2rem;
```

Comment:
you can use only `margin:2rem;` instead of repeating it

Reply:
okay ...will do that..every time i do this same mistake : (